lib/transport_web/templates/dataset/details.html.eex

```diff
@@ -103,27 +103,30 @@
       </div>
 
       <div class="dataset-details__validation">
-        <h2>Validations</h2>
+        <h1><%= dgettext("page-dataset-details", "GTFS Validation") %></h1>
         <%= for {issue_type, issues} <- @dataset.validations do %>
-          <h3><%= issue_type %></h3>
+        <label class="collapse" for="<%= issue_type %>">
+          <h3><%= Gettext.dgettext(TransportWeb.Gettext, "page-dataset-details", issue_type) %> (<%= Enum.count issues %>)</h3>
```

Comment:
> <%= Enum.count issues %>

That should be done *before* it reaches template. Can wait for another PR, however how I see it:

- Add count methods to `reusable_data/dataset.ex`
- Assign them in `reusable_data/reusable_data.ex`

Reply:
is this separation of concerns really worth the boilerplate?

This would require to complexify the structure to have
`%{issue_type => %{issue_count => N, issues => []}`

Reply:
all right… it wasn’t that much boilerplate after all ;)